Atomic.md

```diff
@@ -0,0 +1,190 @@
+<pre>
+Title:       <b>Atomic/Batch Transactions</b>
+Revision:    <b>3</b> (2024-03-14)
+
+Author:      <a href="mailto:mvadari@ripple.com">Mayukha Vadari</a>
+
+Affiliation: <a href="https://ripple.com">Ripple</a>
+</pre>
+
+# Atomic/Batch Transactions
+
+## Abstract
+
+There is a need for transactions to be packaged together and executed atomically.
+
+Potential use cases:
+* Sending an `NFTokenMint` and `NFTokenCreateOffer` transaction in the same fell swoop (if the offer creation fails, so does the mint)
+* Sending a few different offers that might fail and only wanting one to succeed
+* Fees for platforms packaged with the actual transaction
+* Atomic swaps (if multiple accounts are involved)
+
+## 1. Overview
+
+This spec proposes one new transaction: `Atomic`. It will not require any new ledger objects, nor modifications to existing ledger objects. It will require an amendment, tentatively titled `featureAtomic`.
+
+### 1.1. Terminology
+* **Inner transaction**: the sub-transactions included in the `Atomic` transaction, which are the same as the existing set of transactions.
+* **Outer transaction**: the wrapper `Atomic` transaction itself.
+
+## 2. Transaction: `Atomic`
+
+|FieldName | Required? | JSON Type | Internal Type |
+|:---------|:-----------|:---------------|:------------|
+|`TransactionType`|✔️|`string`|`UInt16`|
+|`Account`|✔️|`string`|`STAccount`|
+|`Fee`|✔️|`string`|`STAmount`|
+|`AtomicityType`|✔️|`number`|`UInt8`|
+|`RawTransactions`|!|`array`|`STArray`|
+|`Transactions`|✔️|`array`|`Vector256`|
```

Comment:
Should this instead be called `TxnIDs`, to parallel the naming convention that seems to be implied by [AccountTxnID](https://xrpl.org/docs/references/protocol/transactions/common-fields/#accounttxnid).

Reply:
Good call. I still like `RawTransactions`, but I'm open to changing it.

Reply:
I like `RawTransactions` actually -- or at least something more than just `Transactions` -- as a signal that they won't be stored inside of the Batch Tx the final ledger data.